pkg/config/config.go

```diff
@@ -730,3 +731,31 @@ func (lci *LocalConfigInfo) GetOSSourcePath() (path string, err error) {
 
 	return sourceOSPath, err
 }
+
+// IsValidArgumentList validates the argument list for `odo config` command
+func IsValidArgumentList(args []string) error {
```

Comment:
This function should not be in that file as it only deals with command-related code, not config-related functionality per se. My advice would be to put it with the set command and unexport it.